### ISSUE TITLE
Ability to change page URLs (and for these to automatically redirect)

### ISSUE THREAD

**Issue by joedeville** [COPIM/open-book-collective#90]
E.g. the URL for 'membership' is https://openbookcollective.org/cms/fixed_page/why_support_oa/ (for historical reasons)

**Comment by MartinPaulEve** [comment]
This would require a pretty major rewrite for not much gain. Suggest that you work out what you want the URLs to be and let us know and we can do this as a one-time change.

**Comment by joedeville** [comment]
Yes, ok no problem

**Comment by joedeville** [comment]
List of URLs to be change provided

**Comment by joedeville** [comment]
As per email, suggest changing 'fixed-page' to 'nav'. And there is the list of amended URLs: 
- values TO what_we_stand_for
- why_support_oa TO membership
- what_we_do TO how_it_works
- funding_model TO funding_and_fees

**Comment by MartinPaulEve** [comment]
So to clarify, you want to change, say, "/values/" to "/what_we_stand_for/"?

I would suggest that "what-we-stand-for" would be more common URL formatting (i.e. kebab case rather than snake case).

**Comment by joedeville** [comment]
@MartinPaulEve Yes, correct, and kebab is fine! And what I put down first of all. I just used underscore as that is used elsewhere in the urls at the moment (e.g. fixed_page), but the former makes more sense to me.
